From 156507c6e85ad05256ccba06a710d47d34cd8153 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Fri, 5 May 2017 22:21:19 +0200
Subject: [PATCH] Form: add missing std::vector include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  ../../poppler/Form.h:544:14: error: ‘vector’ in namespace ‘std’ does not name a template type
     const std::vector<Ref> &getCalculateOrder() const { return calculateOrder; }

  ../../poppler/Form.h:556:8: error: ‘vector’ in namespace ‘std’ does not name a template type
     std::vector<Ref> calculateOrder;

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 poppler/Form.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/poppler/Form.h b/poppler/Form.h
index 8ddb6fe..e0a3a28 100644
--- a/poppler/Form.h
+++ b/poppler/Form.h
@@ -27,6 +27,7 @@
 #include "Annot.h"
 
 #include <set>
+#include <vector>
 
 class GooString;
 class Array;
-- 
2.11.0

